test/deploy/validate-configs.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+# Script to validate the packaged module and configs
+
+# Ensure our workdir is that of the project root
+cd "${0%/*}/../.." || exit 
+
+TEST_NAME="Deployment testing"
+declare -a CONFIG_ARRAY
+if [[ $# -gt 0 ]]; then
+    for CONFIG in "$@"; do
+      CONFIG=$(ls {"$CONFIG",config/"$CONFIG",config/"$CONFIG".json,"$CONFIG".json} 2> /dev/null)
+      if [ ! -f "$CONFIG" ]; then
+        echo "Config file $CONFIG does not exist, check the path."
+        exit 1
+      fi
+      CONFIG_ARRAY+=("$CONFIG")
+    done
+    echo "Deployment testing ${#CONFIG_ARRAY[@]} configs:"
+else
+  mapfile -t CONFIG_ARRAY < <(ls config/*.json)
+  echo "Deployment testing all configs:"
+fi
+printf " - %s\n" "${CONFIG_ARRAY[@]}"
+
+mkdir -p .tmp
```

Comment:
Should be in `.gitignore`.

Reply:
Could also just use `/test/tmp` which already is in the `.gitignore` and put both folder (`.tmp` and `.data`) in there.